docs/pull-modes.md

```diff
@@ -0,0 +1,30 @@
+## Pull modes of soci-snapshotter
+
+Soci-snapshotter is a remote snapshotter. It is able to lazily load the contents of a container image when a *SOCI index* is present in the remote registry. If a SOCI index is not found, it will download and uncompress the layers at launch time, just like the default snapshotter does. SOCI indices can also be "sparse", meaning that any individual layer may not be indexed. In that case, that layer will be downloaded at launch time, while the indexed layers will be lazily loaded.
```

Comment:
Why not be more specific just saying overlay snapshotter not " default snapshotter" which I imagine would raise a question of what is the default?

Reply:
Soci-snapshotter does not define which snapshotter will be used for non-soci layers. It downloads/decompresses the layer blob and then gives control to container runtime. That said, the "default snapshotter" should be a better definition here.